code/modules/ninja/suit/mask.dm

```diff
@@ -10,130 +10,9 @@ Contents:
 
 
 
-/obj/item/clothing/mask/gas/voice/space_ninja
+/obj/item/clothing/mask/gas/space_ninja
 	name = "ninja mask"
 	desc = "A close-fitting mask that acts both as an air filter and a post-modern fashion statement."
 	icon_state = "s-ninja"
 	item_state = "s-ninja_mask"
-	vchange = 1
 	strip_delay = 120
-
-/obj/item/clothing/mask/gas/voice/space_ninja/speechModification(message)
```

Comment:
oh yeah I should probably put that in the changelog

space ninja snowflake was too retarded for me to bother keeping and the person who redid it said they didn't care anymore/didn't give a response when I asked if they cared at all, so I axed it. No reason to keep it in since literally no one uses it and admins haven't spawned a ninja in literally fucking ages.

Then again, saying this is gonna make an admin go spawn a ninja and then pop out of the woodwork saying they make ninjas "all the time", so  ¯\_(ツ)_/¯